File: dotnetcore/StarterBot/Entities/GameState.cs
using System.Collections.Generic;

namespace StarterBot.Entities
{
    public class GameState
    {
        public int CurrentRound { get; set; }
        public int MaxRounds { get; set; }
        public int MapSize { get; set; }
        public int ConsecutiveDoNothingCount { get; set; }

        public Player Player { get; set; }

        public Lane[] Lanes { get; set; }
    }
}

File: reference-bot/java/src/main/java/za/co/entelect/challenge/entities/GameState.java
package za.co.entelect.challenge.entities;

import com.google.gson.annotations.SerializedName;
import za.co.entelect.challenge.enums.PowerUps;
import za.co.entelect.challenge.enums.State;

import java.util.List;

public class GameState {

    @SerializedName("currentRound")
    public int currentRound;

    @SerializedName("maxRounds")
    public int maxRounds;

    @SerializedName("player")
    public Car player;

    @SerializedName("opponent")
    public Car opponent;

    @SerializedName("worldMap")
    public List<Lane[]> lanes;

}
